### PR TITLE
Idp localization - part 2

### PR DESCRIPTION
**Pull request by vihanga-liyanage**

### CODE REVIEW

File: apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx [hunk 1]
```diff
@@ -152,21 +152,34 @@ export const initSubjectAndRoleURIs = (initialClaims, setSubjectClaimUri, setRol
...
-            dispatch(addAlert({
-                description: "An error occurred while the updating claims configurations.",
+        .catch((error) => {
+            if (error.response && error.response.data && error.response.data.description) {
+                store.dispatch(addAlert({
+                    description: I18n.instance.t("devPortal:components.idp.notifications." +
+                        "updateClaimsConfigs.error.description",
+                        { description: error.response.data.description }),
+                    level: AlertLevels.ERROR,
+                    message: I18n.instance.t("devPortal:components.idp.notifications.updateClaimsConfigs.error.message")
```
Comment (brionmario) [inline apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx:173]: Possible line length issue here. Please check.

File: apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx [hunk 2]
```diff
@@ -175,21 +188,34 @@ export const handleAttributeSettingsFormSubmit = (idpId: string, values: Identit
...
     ).then(() => {
-        dispatch(addAlert(
+        store.dispatch(addAlert(
             {
-                description: "Successfully updated role mapping configurations.",
+                description: I18n.instance.t("devPortal:components.idp.notifications." +
+                    "updateIDPRoleMappings.success.description"),
                 level: AlertLevels.SUCCESS,
-                message: "Update successful"
+                message: I18n.instance.t("devPortal:components.idp.notifications.updateIDPRoleMappings.success.message")
```
Comment (brionmario) [inline apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx:196]: Possible line length issue here. Please check.

File: apps/developer-portal/src/components/identity-providers/wizards/outbound-provisioning-connector-create-wizard.tsx [hunk 3]
```diff
@@ -16,30 +16,34 @@
  * under the License.
  */
 
+import { AlertLevels } from "@wso2is/core/dist/src/models";
```
Comment (brionmario) [inline apps/developer-portal/src/components/identity-providers/wizards/outbound-provisioning-connector-create-wizard.tsx:19]:
```suggestion
import { AlertLevels } from "@wso2is/core/models";
```

File: apps/developer-portal/src/components/identity-providers/wizards/outbound-provisioning-connector-create-wizard.tsx [hunk 4]
```diff
@@ -16,30 +16,34 @@
...
+import { addAlert } from "@wso2is/core/dist/src/store";
```
Comment (brionmario) [inline apps/developer-portal/src/components/identity-providers/wizards/outbound-provisioning-connector-create-wizard.tsx:20]:
```suggestion
import { addAlert } from "@wso2is/core/store";
```

File: apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx (hunk 1 above)
Comment (vihanga-liyanage) [reply apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx:173]: Not according to Webstorm.

File: apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx (hunk 2 above)
Comment (vihanga-liyanage) [reply apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx:196]: Not according to Webstorm.

**Pull request merged**